src/strings/en-us.json

```diff
@@ -15,7 +15,7 @@
     "Alerts": "Alerts",
     "All": "All",
     "AllChannels": "All channels",
-    "AllComplexFormats": "All complex formats (ASS, SSA, VOBSUB, PGS, SUB/IDX, etc.)",
+    "AllComplexFormats": "All Complex Formats (ASS, SSA, VOBSUB, PGS, SUB, IDX)",
```

Comment:
why do you edit language files when adding configuration? don't mix unrelated stuff please :)

Reply:
I made these changes because I was originally adding some configuration options to the subtitles options page, I wouldn't normally do that.

Reply:
I'll believe you... this time 👅